plugins/eksporter.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: eksporter
+spec:
+  version: "v1.0.0"
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
```

Comment:
The plugin is a ruby script. Shouldn't a windows environment with a ruby installation not also qualify as a target OS?

Reply:
Fixd, and will be in the next release after I've updated the descriptions and help text.